src/components/_overlays/MemberDropdown/MemberDropdown.tsx

```diff
@@ -0,0 +1,204 @@
+  const [isSwitchingMember, setIsSwitchingMember] = useState(false)
+  const navigate = useNavigate()
+  const { activeChannelId, activeMembership, setActiveUser, memberships } = useUser()
+
+  const hasOneMember = memberships?.length === 1
+
+  const handleAddNewMember = () => {
+    navigate(absoluteRoutes.studio.newMembership())
+  }
+  const handleSwitchMemberMode = () => {
+    setIsSwitchingMember(true)
+  }
+  const handleAddNewChannel = () => {
+    navigate(absoluteRoutes.studio.newChannel())
+  }
+  const handleGoToJoystream = () => {
+    navigate(absoluteRoutes.viewer.index())
+  }
+  const handleGoToStudio = () => {
+    navigate(absoluteRoutes.studio.index())
+  }
+  const handleGoToMyProfile = () => null
+  const handleChannelChange = (channelId: string) => {
+    setActiveUser({ channelId })
+  }
+  const handleMemberChange = (memberId: string) => {
+    setActiveUser({ memberId })
+    setIsSwitchingMember(false)
+  }
+
+  return (
+    <Container>
+      {isSwitchingMember ? (
+        <div>
+          <SwitchMemberItemListContainer>
+            <ListItem
+              onClick={() => setIsSwitchingMember(false)}
+              nodeStart={<StyledSvgActionChevronL />}
+              label="Switch member"
+            />
+          </SwitchMemberItemListContainer>
+
+          <SectionContainer>
+            {memberships?.map((member) => (
+              <ListItem
+                key={member.id}
+                onClick={() => handleMemberChange(member.id)}
+                nodeStart={<Avatar assetUrl={member.avatarUri} />}
+                label={member.handle ?? ''}
+                selected={member.id === activeMembership?.id}
+              />
+            ))}
+            <ListItem
+              nodeStart={<IconWrapper icon={<SvgActionNewChannel />} />}
+              onClick={() => handleAddNewMember()}
+              label={'Add new member...'}
+            />
+          </SectionContainer>
+        </div>
+      ) : (
+        <div>
+          <BlurredBG url={activeMembership?.avatarUri}>
+            <Filter />
+            <MemberInfoContainer>
+              <StyledAvatar size="fill" assetUrl={activeMembership?.avatarUri}></StyledAvatar>
+              <div>
+                <Text variant="h400">{activeMembership?.handle}</Text>
+                <TjoyContainer>
+                  <BalanceContainer>
+                    <SvgActionJoyToken />
+                    <Text variant="t200-strong">12.5K</Text>
+                  </BalanceContainer>
+                  <Divider />
+                  <Text
+                    variant="t100"
+                    as="a"
+                    // @ts-ignore our types don't allow this but its fine here
+                    href="https://www.joystream.org/"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    secondary
+                    color={cVar('colorCoreNeutral200Lighten')}
+                  >
```

Comment:
In the design `Learn about tJOY` isn't underlined. 
![image](https://user-images.githubusercontent.com/51168865/146331489-d604e4ce-d792-4135-bd3f-83638bb5ab65.png)
But to be honest I like underlined text here. It's suggesting that it's a clickable link. Could you ask @toiletgranny if we could leave this without change as you have here?

Reply:
btw. I'm fine with @ts-ignore

Reply:
@toiletgranny what do you think?

Reply:
Talked to Adam about this, and he decided that it should be underlined only when you're hovering over this link.